config.js

```diff
@@ -824,6 +824,10 @@ config.NSFS_LOW_FREE_SPACE_MB_UNLEASH = 10 * 1024;
 // operations safely.
 config.NSFS_LOW_FREE_SPACE_PERCENT_UNLEASH = 0.10;
 
+// anonymous account name
+config.NC_ANON_ACCOUNT_NAME = 'anonymous'
+config.ANONYMOUS_ACCESS_KEY = Symbol('anonymous_access_key');
```

Comment:
As a config this can be overridden, which is not really meant to be used.
I would prefer to have this as a const in object_sdk, and export it to be shared for bucketspace_fs.